tools/cleanReviewerInterests.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * @file tools/cleanReviewerInterests.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ReviewerInterestsDeletionTool
+ * @ingroup tools
+ *
+ * @brief CLI tool to remove user interests that are not referenced by any user accounts. 
+ */
+
+require(dirname(__FILE__) . '/bootstrap.inc.php');
+
+class ReviewerInterestsDeletionTool extends CommandLineTool {
+
+	/**
+	 * Constructor.
+	 * @param $argv array command-line arguments
+	 */
+	public function __construct($argv = array()) {
+		parent::CommandLineTool($argv);
```

Comment:
this is because I had branched off ojs-stable-3_0_0. I will send another fix in a bit